### ISSUE TITLE
`modluafooter` doesn't place lines at the end of the file

### ISSUE THREAD

**Issue by kayleanelson** [easybuilders/easybuild-framework#3002]
`modluafooter` name suggests that it will place the lines after all other modules lines have been written (aka in the footer), but this is not true. Module lines written by a custom easyblock show up after the lines from `modluafooter`. It would be preferable for these lines to reliably be the last lines (other than the EasyBuild comments). 

In our particular use case, we would like to be able to unset an environment variable set by an EasyBlock. Example:

```
unsetenv("_JAVA_OPTIONS") -- written by modluafooter
prepend_path("PATH", root)
setenv("_JAVA_OPTIONS", "-Xmx256m")
-- Built with EasyBuild version 3.9.3
```

**Comment by boegel** [comment]
Thanks for the bug report @kayleanelson!

This is pretty easy to fix, since we already have `make_module_footer` method, where picking up `modluafooter` clearly belong...

Fixed in #3003, should be included with the upcoming EasyBuild v4.0.0

**Comment by boegel** [comment]
fixed in #3003
